server/main.go

```diff
@@ -279,26 +270,36 @@ func main() {
 	var pprofUrl = flag.String("pprof_url", "", "Debugging only! URL path for exposing profiling info. Disabled if not set.")
 	flag.Parse()
 
+	logs.Init(os.Stderr, *logFlags)
```

Comment:
The first parameter `os.Stderr` is not used.

Reply:
Fixed.